spring-auto-configuration/src/main/java/com/epam/autoconfig/config/DataSourceConfig.java

```diff
@@ -0,0 +1,20 @@
+package com.epam.autoconfig.config;
+
+import org.springframework.context.annotation.Bean;
+import org.springframework.context.annotation.ComponentScan;
+import org.springframework.context.annotation.ComponentScans;
```

Comment:
don't forget to delete unused imports (you can use ctrl+alt+o)

Reply:
@setyartan done